### PR TITLE
GHG protocol-Refrigerants

### PR DESCRIPTION
**Pull request by maryarbab**
Is this a new addition or an update to an existing factor?

[*] New

What category or categories does it affect?

Refrigerants and Fugitive Gases

What?

Adding GHG protocol EFs along with the constituent gasses and scope as new columns.

Why?

To grow the scope and coverage of the OEFDB.

### CODE REVIEW

File: OpenEmissionFactorsDB.csv [hunk 1]
```diff
@@ -1644,67 +1644,137 @@ Organizational Activities,Wholesale Trade,wholesale_trade-type_petroleum_petrole
...
+Refrigerants and Fugitive Gases,Refrigerants and Fugitive Gases,fugitive_gas-type_carbon_dioxide,Carbon dioxide,not-supplied,1,1,not-supplied,not-supplied,not-supplied,not-supplied,not-supplied,1,12,GHG Protocol,2021,GLOBAL,2022/01/15,Emission intensity for carbon dioxide. This conversion factor should be used to report leakage from air-conditioning refrigeration units or the release to the atmosphere of other gases that have global warming potential (GWP). The primary sources are IPCC Fourth Assessment Report (AR4) - GWPs: IPCC's Working Group I - Climate Change 2007: The Physical Science Basis. Ch. 2 - p. 212-215 - Tables 2.14 and 2.15 and IPCC Fifth Assessment Report (AR5) - GWPs: IPCC's Working Group I - Climate Change 2013: The Physical Science Basis. Ch. 8 - p. 731-738 - Table 8.A.1.,https://ghgprotocol.org/ghg-emissions-calculation-tool,unknown
```
Comment (AnastasiaLobanova) [inline OpenEmissionFactorsDB.csv]: This is really cool description - with references to all tables in the AR4 and AR5! I was wondering if it is too detailed and we can just specify the report and the user could find the GWPs him/herself?

File: OpenEmissionFactorsDB.csv [hunk 2]
```diff
@@ -1713,8 +1783,81 @@ Refrigerants and Fugitive Gases,Refrigerants and Fugitive Gases,fugitive_gas-typ
...
+Refrigerants and Fugitive Gases,Refrigerants and Fugitive Gases,fugitive_gas-type_pfc_116,PFC-116 (Perfluoroethane)  ,not-supplied,"11,100","12,200",not-supplied,not-supplied,not-supplied,not-supplied,not-supplied,1,12,GHG Protocol,2021,GLOBAL,2022/01/15,Emission intensity for PFC-116 (Perfluoroethane). This conversion factor should be used to report leakage from air-conditioning refrigeration units or the release to the atmosphere of other gases that have global warming potential (GWP). The primary sources are IPCC Fourth Assessment Report (AR4) - GWPs: IPCC's Working Group I - Climate Change 2007: The Physical Science Basis. Ch. 2 - p. 212-215 - Tables 2.14 and 2.15 and IPCC Fifth Assessment Report (AR5) - GWPs: IPCC's Working Group I - Climate Change 2013: The Physical Science Basis. Ch. 8 - p. 731-738 - Table 8.A.1.,https://ghgprotocol.org/ghg-emissions-calculation-tool,unknown
```
Comment (AnastasiaLobanova) [inline OpenEmissionFactorsDB.csv]: Here (and below, until line 1792) are some extra white spaces in the name column

**Review by AnastasiaLobanova — DISMISSED** [review]
Great work Maryam! We need to change the number formatting and I would shorten a bit the description! Thank youuu

File: OpenEmissionFactorsDB.csv [hunk 3]
```diff
@@ -1644,67 +1644,137 @@ Organizational Activities,Wholesale Trade,wholesale_trade-type_petroleum_petrole
...
+Refrigerants and Fugitive Gases,Refrigerants and Fugitive Gases,fugitive_gas-type_carbon_tetrachloride,Carbon tetrachloride,not-supplied,"1,730","1,400",not-supplied,not-supplied,not-supplied,not-supplied,not-supplied,1,12,GHG Protocol,2021,GLOBAL,2022/01/15,Emission intensity for carbon tetrachloride. This conversion factor should be used to report leakage from air-conditioning refrigeration units or the release to the atmosphere of other gases that have global warming potential (GWP). The primary sources are IPCC Fourth Assessment Report (AR4) - GWPs: IPCC's Working Group I - Climate Change 2007: The Physical Science Basis. Ch. 2 - p. 212-215 - Tables 2.14 and 2.15 and IPCC Fifth Assessment Report (AR5) - GWPs: IPCC's Working Group I - Climate Change 2013: The Physical Science Basis. Ch. 8 - p. 731-738 - Table 8.A.1.,https://ghgprotocol.org/ghg-emissions-calculation-tool,unknown
```
Comment (patstirling) [inline OpenEmissionFactorsDB.csv]: remove commas from these and any other gas conversion numbers

File: OpenEmissionFactorsDB.csv [hunk 4]
```diff
@@ -1644,67 +1644,137 @@ Organizational Activities,Wholesale Trade,wholesale_trade-type_petroleum_petrole
...
+Refrigerants and Fugitive Gases,Refrigerants and Fugitive Gases,fugitive_gas-type_acetaldehyde,Acetaldehyde,not-supplied,not-supplied,1,not-supplied,not-supplied,not-supplied,not-supplied,not-supplied,1,12,GHG Protocol,2021,GLOBAL,2022/01/15,Emission intensity for acetaldehyde. This conversion factor should be used to report leakage from air-conditioning refrigeration units or the release to the atmosphere of other gases that have global warming potential (GWP). The primary sources are IPCC Fourth Assessment Report (AR4) - GWPs: IPCC's Working Group I - Climate Change 2007: The Physical Science Basis. Ch. 2 - p. 212-215 - Tables 2.14 and 2.15 and IPCC Fifth Assessment Report (AR5) - GWPs: IPCC's Working Group I - Climate Change 2013: The Physical Science Basis. Ch. 8 - p. 731-738 - Table 8.A.1.,https://ghgprotocol.org/ghg-emissions-calculation-tool,unknown
```
Comment (patstirling) [inline OpenEmissionFactorsDB.csv]: unit column should be "kg" throughout

File: OpenEmissionFactorsDB.csv [hunk 5]
```diff
@@ -1730,7 +1873,11 @@ Refrigerants and Fugitive Gases,Refrigerants and Fugitive Gases,fugitive_gas-typ
...
+Refrigerants and Fugitive Gases,Refrigerants and Fugitive Gases,fugitive_gas-type_trichloromethane,Trichloromethane (Chloroform),not-supplied,16,not-supplied,not-supplied,not-supplied,not-supplied,not-supplied,not-supplied,,12,GHG Protocol,2021,GLOBAL,2022/01/15,Emission intensity for trichloromethane (Chloroform). This conversion factor should be used to report leakage from air-conditioning refrigeration units or the release to the atmosphere of other gases that have global warming potential (GWP). The primary sources are IPCC Fourth Assessment Report (AR4) - GWPs: IPCC's Working Group I - Climate Change 2007: The Physical Science Basis. Ch. 2 - p. 212-215 - Tables 2.14 and 2.15 and IPCC Fifth Assessment Report (AR5) - GWPs: IPCC's Working Group I - Climate Change 2013: The Physical Science Basis. Ch. 8 - p. 731-738 - Table 8.A.1.,https://ghgprotocol.org/ghg-emissions-calculation-tool,unknown
```
Comment (patstirling) [inline OpenEmissionFactorsDB.csv]: This line missing scope

**Review by patstirling — CHANGES_REQUESTED** [review]

File: OpenEmissionFactorsDB.csv (hunk 1 above)
Comment (patstirling) [reply OpenEmissionFactorsDB.csv]: Agree - think we can just say "The primary sources are IPCC Fourth Assessment Report (AR4) and IPCC Fifth Assessment Report (AR5)" here

**Review by patstirling — APPROVED** [review]

**Pull request merged**